src/lib/curl/base.c

```diff
@@ -29,7 +29,16 @@
 #include <curl/curl.h>
 #include <talloc.h>
 
-static uint32_t instance_count = 0;
+
+
+static 			int 		instance_count;
```

Comment:
There's no reason to add this whitespace or change instance count back to an `int`.